web/src/components/UserProfile/UserProfile.js

```diff
@@ -91,6 +91,10 @@ const UserProfile = ({ user, isEditable, loading, onSave, error }) => {
             />
           </div>
         </div>
+        <div className="mt-10">
+          <h3 className="text-3xl text-gray-500 font-ropa-sans">Parts:</h3>
+          <PartsCell />
```

Comment:
I didn't realise this was going to be this simple. Really elegant solution well done. You've clearly found your way around the repo ✨